packages/oauth2/src/index.ts

```diff
@@ -4,5 +4,14 @@ export * from "./errors";
 export * from "./pkce";
 export * from "./token-dpop";
```

Comment:
```suggestion
export * from "./token-dpop";
export * from "./client-attestation-pop"
```

I think the export of the `client-attestation-pop`'s functionalities is missing.